src/main/java/org/eln2/mc/common/cell/ElectricalComponentConnection.kt

```diff
@@ -0,0 +1,6 @@
+package org.eln2.mc.common.cell
+
+import org.ageseries.libage.sim.electrical.mna.component.Component
+
+// TODO: This should be abstracted by the relevant simulator with a generic to allow for more than just a Electrical component.
+data class ElectricalComponentConnection(val component: Component, val index : Int)
```

Comment:
I renamed this to be more representative of what it is; in the future we should abstract this per MNA and then the mna will register a new subclass that it uses for transport of the feature.

This interface should probably come from libsim or libsim-minecraft to allow for MNA's to prepare for that sort of inbound structure?